extensions/podman/src/extension.ts

```diff
@@ -432,11 +438,13 @@ async function registerProviderFor(provider: extensionApi.Provider, machineInfo:
       await startMachine(provider, machineInfo, context, logger);
     },
     stop: async (context, logger): Promise<void> => {
-      await execPromise(getPodmanCli(), ['machine', 'stop', machineInfo.name], { context, logger });
+      await extensionApi.process.exec(getPodmanCli(), ['machine', 'stop', machineInfo.name], {
+        logger: new LoggerDelegator(context, logger),
```

Comment:
Could you explain why we need a 'LoggerDelegator'
I would expect that we won't need 3rd party utilities

Reply:
added description to the LoggerDelegator class: https://github.com/containers/podman-desktop/pull/3315/files#diff-62f1e4c4e882bb44ccd1b237c25717b7bab687b6e4f2fbe41576294a555209f4R63